docs/getting-started.md

```diff
@@ -7,283 +7,212 @@ path: /docs/getting-started/
 
 # Getting Started
 
-This guide will help you get started using MDC Web on your own sites and within your own projects.
+This getting started guide walks you through how to [install MDC Web Node modules](https://www.npmjs.com/org/material), and bundle the Sass and JavaScript from those Node modules in your [Webpack](https://webpack.js.org/) configuration
 
-> If you are interested in integrating MDC Web into a framework, or building a component library for
-your framework that wraps MDC Web, check out our [framework integration guide](./integrating-into-frameworks.md).
+> Note: This guide assumes you have npm installed locally.
 
-## MDC Web quick start: building a simple greeting app
+### Step 1: Webpack with Sass
 
-The best way to learn any new technology is to get your hands dirty and build something with it, so
-that is what we will do here!  You will be building a simple greeting page which lets you enter a name and greets you as such.
+We’re going to use webpack-dev-server to demonstrate how webpack bundles our Sass and Javascript. First create a package.json that looks like this:
 
-As you go through this guide, we encourage you to code along with it. By the end, you will have
-learned the fundamentals incorporating MDC Web into simple sites, as well as worked with some of the
-components we have to offer.
+```
+{
+  "scripts": {
+    "start": "webpack-dev-server"
+  }
+}he
+```
 
-### Setting up the project
+You’ll need all of these Node dependencies:
+- [webpack](https://www.npmjs.com/package/webpack): Bundles Sass and JavaScript
+- [webpack-dev-server](https://www.npmjs.com/package/webpack-dev-server): Development server
+- [sass-loader](https://www.npmjs.com/package/sass-loader): Loads a Sass file and compiles it to CSS
+- [node-sass](https://www.npmjs.com/package/node-sass): Provides binding for Node.js to Sass, peer dependency to sass-loader
+- [css-loader](https://www.npmjs.com/package/css-loader): Resolves CSS @import and url() paths
+- [extract-loader](https://github.com/peerigon/extract-loader): Extracts the CSS into a .css file
+- [file-loader](https://github.com/webpack-contrib/file-loader): Serves the .css file as a public URL
 
-Create a directory for the project where we'll serve our application out of.
+You can install all of them by running this command:
 
 ```
-mkdir greeting-app
-cd greeting-app
+npm install --save-dev webpack@3.0.0 webpack-dev-server@2.4.3 css-loader sass-loader node-sass extract-loader file-loader
```

Comment:
`webpack@3` should install the highest 3.x (currently 3.11.0) which is probably a better idea than pinning the first ever release of version 3. Ditto `webpack-dev-server@2` (or `@2.4` if we're pinning to that version for some reason...)